src/sentry/search/snuba/backend.py

```diff
@@ -60,13 +59,13 @@ def calculate(data):
         'priority', ['last_seen', 'times_seen'], calculate_priority_cursor,
     ),
     'date': (
-        'last_seen', [], _datetime_cursor_calculator('last_seen', max),
+        'last_seen', [], _datetime_cursor_calculator('last_seen'),
```

Comment:
Just a thought, these datetime strings should sort lexicographically? Like maybe we don't need to covert them to sort by them?

Reply:
The only issue is we put that score into the cursor that round trips to the user's request. Keepinging everything an `int` seems simpler (and maintains compatibility for old requests, I guess?).

Reply:
ah, yep definitely, I missed the fact that it was used for the cursor.